scripts/dts/extract_dts_includes.py

```diff
@@ -641,7 +641,12 @@ def generate_include_file(defs, args):
         sys.stdout.write('/* ' + node.split('/')[-1] + ' */')
         sys.stdout.write("\n")
 
-        maxlength = max(len(s + '#define ') for s in defs[node].keys())
+        max_dict_key = lambda d: max(len(k) for k in d.keys())
```

Comment:
Hmmm.. is this part of your patch?

Reply:
@dbkinder can you please clarify your question?

per the commit message: "If there's any key in the alias which length is larger than other keys
in the node, the include file will be incorrect, there will be no tab
between the key and value.

We need to take into account the max length of alias keys."

Reply:
I wasn't sure that the changes to scripts/dts/extract_dts_includes.py were actualy part of PR #1502 for get/settimeofday

Reply:
When reviewing PRs it helps to review patches individually instead of the entire diff of the patch series. This change was documented in the commit message.

Reply:
OK, but still unclear to me why there's a patch to extract_dts_includes.py mixed in with patches to a PR titled "Support gettimeofday() and settimeofday() ".  Shouldn't the patch to extract_dts_includes.py be in a separate PR?

Reply:
Have a look at the patch series in order.
First patch fixes issues in DTS script.
Second patch introduces DTS for RTC driver
Third patch implements the RTC driver.
Seems to me that the script change is a dependency for the RTC driver DTS.

Reply:
Ah. I understand the flow now.  We've had some problems where extraneous commits were showing up in PRs so I thought this might be another. Thanks.

Reply:
@dbkinder, I encounter the issue that the generated_dts_board.h is incorrect when I modify following changes to dts file.

```
 +&rtc {
 +       status = "ok";
 +       interrupts = <46 0>;
 +       interrupt-names = "alarm";
 +};
```
 
generated_dts_board.h:
```
#define NXP_K64F_RTC_4003D000_IRQ_ALARM_PRIORITYNXP_K64F_RTC_4003D000_IRQ_0_PRIORITY
```
You can see there's no tap between the macro and the definition. 385b340 fixes it and it's necessary to this PR.